starlite/controller.py

```diff
@@ -24,14 +25,21 @@ class Controller:
     guards: Optional[List[Guard]]
 
     def __init__(self, owner: "Router"):
-        if not hasattr(self, "path") or not self.path:
+        if not hasattr(self, "path") or self.path is None:
             raise ImproperlyConfiguredException("Controller subclasses must set a path attribute")
 
         for key in ["dependencies", "response_headers", "response_class", "guards"]:
             if not hasattr(self, key):
                 setattr(self, key, None)
 
-        self.path = normalize_path(self.path)
+        if self.path == "":
+            warnings.warn(
+                "Empty controller's path. Routes will be added to the application without prefix.",
+                UserWarning,
+            )
+        else:
+            self.path = normalize_path(self.path)
```

Comment:
If the path is empty we don't need to use `normalize_path`. (in `normalize_path` it will first add `/` and then remove `/`).